internal/storage/fs/object/s3/store.go

```diff
@@ -2,18 +2,43 @@ package s3
 
 import (
 	"context"
+	"fmt"
+	"net/url"
 	"sync"
 	"time"
 
-	"github.com/aws/aws-sdk-go-v2/config"
-	"github.com/aws/aws-sdk-go-v2/service/s3"
-
+	s3v2 "github.com/aws/aws-sdk-go-v2/service/s3"
 	"go.flipt.io/flipt/internal/containers"
 	"go.flipt.io/flipt/internal/storage"
 	storagefs "go.flipt.io/flipt/internal/storage/fs"
+	"go.flipt.io/flipt/internal/storage/fs/object/blob"
 	"go.uber.org/zap"
+	gcaws "gocloud.dev/aws"
+	gcblob "gocloud.dev/blob"
+	"gocloud.dev/blob/s3blob"
+)
+
+const (
+	Schema = "s3i"
```

Comment:
is this a typo? should it be `s3`?

Reply:
No, it isn't. For some reason, Flipt uses `UsePathStyle=true` option to access AWS S3.  Using `go-cloud s3` will introduce a breaking change.  `go-cloud` doesn't provide the easy way to configure that option for aws sdk v2. There is `s3ForcePathStyle` for aws sdk v1 but `v1` has hardcoded AWS regions and there is no way to use `minio` as a region with it.

So I have to create a new schema with that name that will match previous behaviour. 

https://github.com/erka/flipt/blob/d9f0900db0b9e4875e80cc21b205f557ce8db30a/internal/storage/fs/object/s3/store.go#L29-L40

Reply:
Ah I see now. Thank you for the explanation!

Reply:
This is a requirement to make minio work nicely with the s3 SDK in the integration tests. (UsePathStyle). For reference.